cypress/e2e/recommendation-page.cy.js

```diff
@@ -0,0 +1,53 @@
+describe('Recommendation page flow', () => {
+
+  beforeEach(() => {
```

Comment:
Not sure if it's been tried but this article goes through the entire issue and may have a solution chrome geo location access: https://glebbahmutov.com/blog/cypress-geolocation/